include/swift/IDE/Utils.h

```diff
@@ -176,6 +176,11 @@ struct ResolvedValueRefCursorInfo : public ResolvedCursorInfo {
       llvm::None;
 
   bool IsKeywordArgument = false;
+  /// Indicates that this cursor info is resolved for an argument name
+  /// and not the parameter name. \c ValueD always points to the surrounding
+  /// param declaration because the argument name is not a standalone
+  /// declaration.
+  bool IsExtArgName = false;
```

Comment:
I always get confused with the argument vs. parameter name. Just to clarify, could you add an example here? For example, the following (and please double-check that it’s indeed correct).

```cpp
///
/// E.g. when having the parameter declaration 'first second: Int' and renaming \c first, then \c IsExtArgName is \c true and when renaming \c second, then \c IsExtArgName is \c false.
```